components/utils/src/templates.rs

```diff
@@ -100,6 +100,16 @@ pub fn render_template(
     }
 }
 
+pub fn is_default_template(name: &str, tera: &Tera, theme: &Option<String>) -> Result<bool> {
```

Comment:
That's not the right place to check. You essentially need to search for templates with that name in a theme or the templates directory Tera instances. Only if there are no overrides should the message be shown. Another way to do it would be to have a `log` Tera functions that does println so the template itself can warn

Reply:
> You essentially need to search for templates with that name in a theme or the templates directory Tera instances. Only if there are no overrides should the message be shown. 

Sorry, I thought I had accomplished this with this function which uses check_template_fallbacks.  I will think on this but I am currently confused as to how to do this differently without repeating the logic in check_template_fallbacks.

Reply:
How do I do this without repeating logic in check_template_fallbacks?